### PR TITLE
Organization Email Signup

### PR DESCRIPTION
**Pull request by crchong1**

### CODE REVIEW

File: src/test/UserTest/UserControllerIntegrationTest.java [hunk 1]
```diff
@@ -51,4 +49,34 @@ public void loginUserWithNoUsernameTest() {
...
+    body.put("personRole", "Worker");
+    body.put("orgName", "");
+
+    HttpResponse<String> actualResponse =
+        Unirest.post(TestUtils.getServerUrl() + "/create-invited-user")
+            .body(body.toString())
+            .asString();
+
+    JSONObject actualResponseJSON = TestUtils.responseStringToJSON(actualResponse.getBody());
+    System.out.println(actualResponseJSON);
```
Comment (crchong1) [inline src/test/UserTest/UserControllerIntegrationTest.java:77]: I would remove the print line here

File: src/test/UserTest/UserControllerIntegrationTest.java [hunk 2]
```diff
@@ -51,4 +49,34 @@ public void loginUserWithNoUsernameTest() {
...
+
+    assert (actualResponseJSON.has("status"));
+    assertThat(actualResponseJSON.getString("status")).isEqualTo("INVALID_PARAMETER");
```
Comment (crchong1) [inline src/test/UserTest/UserControllerIntegrationTest.java:80]: is there any way we can add other cases? a test that gives a SUCCESS status is essential

**Review by crchong1 — COMMENTED** [review]
@MelindaCardenas I left a couple comments :)

**Review by crchong1 — COMMENTED** [review]
everything looks good- @MelindaCardenas you can merge if you are ready

**Review by MelindaCardenas — APPROVED** [review]

**Pull request merged**